### PR TITLE
DI-358 Add org type info  to event replay logs

### PR DESCRIPTION
**Pull request by rohith1122**
## Link to JIRA Ticket

- https://nhsd-jira.digital.nhs.uk/browse/DI-358

## Description

As per the current architecture dentist changed events will be stored as normal like pharmacy changed events for DLQ and replay purpose

### Noteworthy Changes

- Added missing org_type to event replay logs

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing

Please tick the testing that has been completed

- [ ] Unit tests
- [ ] Integration tests

## Developer Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have run the [code formatting checks](../README.md#code-quality)
- [ ] I have run the [code quality checks](../README.md#code-quality)
- [ ] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [ ] Unit test code coverage is at or above 80%
- [ ] New and existing unit tests pass locally with my changes
- [ ] Tests have added that prove my fix is effective or that my feature works (Integration tests)
- [ ] I have made corresponding changes to the documentation
- [ ] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [ ] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

File: application/event_replay/event_replay.py [hunk 1]
```diff
@@ -32,8 +32,10 @@ def lambda_handler(event: Dict[str, Any], context: LambdaContext) -> str:
     logger.set_correlation_id(correlation_id)
     validate_event(event)
     odscode = event["odscode"]
+    org_type = event["org_type"]
```
Comment (JackPlowman) [inline application/event_replay/event_replay.py:35]: Are you expecting the user to pass in the org_type_id?

Comment (JackPlowman) [reply application/event_replay/event_replay.py:35]: If so there needs to be documentation changes on confluence. Or you get the organisation type id from the change event in dynamodb

**Review by JackPlowman — CHANGES_REQUESTED** [review]

File: application/event_replay/event_replay.py (hunk 1 above)
Comment (rohith1122) [reply application/event_replay/event_replay.py:35]: No, I am expecting org_type from the event object

**Review by ed-sparkes — APPROVED** [review]

**Review by JackPlowman — APPROVED** [review]
LGTM 👍

**Review by Corbisc — APPROVED** [review]
All looks good here.

**Pull request merged**